scene/2d/camera_2d.cpp

```diff
@@ -748,7 +731,7 @@ void Camera2D::_bind_methods() {
 
 	ADD_GROUP("Smoothing", "smoothing_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "smoothing_enabled"), "set_enable_follow_smoothing", "is_follow_smoothing_enabled");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "smoothing_speed", PROPERTY_HINT_NONE, "suffix:px/s"), "set_follow_smoothing", "get_follow_smoothing");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "smoothing_speed", PROPERTY_HINT_RANGE, "0,60,1"), "set_smoothing_speed", "get_smoothing_speed");
```

Comment:
What happened to the suffix?

Reply:
I think @madmiraal says in the notes that the original suffix was incorrect, therefore the removal.

Reply:
Yes, the docs were updated with the use of `pixels per second` in 1063ddfeb52c796a3207a62142cc3b7ae3fbccb7, but as pointed out by @Calinou in https://github.com/godotengine/godot/issues/49747#issuecomment-864444744:
> The camera's speed cannot be defined in pixels per second because the number of pixels second travelled will depend on how far the camera is from the target. Camera smoothing uses asymptotic smoothing, not linear interpolation.

The docs have been updated with:
> ... the asymptotic speed at which the camera closes the gap between the current position and the target position.